### PR TITLE
production frame and confirm production changed

### PR DESCRIPTION
**Pull request by liorama**
I've formatted the HTML file so added comments where there were changes

### CODE REVIEW

File: scripts/src/admin/wrapper.html [hunk 1]
```diff
@@ -1,492 +1,510 @@
...
+					updateConfig(editor)
+				}
+				break;
+			case 27:
+				closeEditor(editor);
+				break;
+		}
+	}
+
+	function confirmCbIfProduction(cb) {
```
Comment (liorama) [inline scripts/src/admin/wrapper.html]: This is new

File: scripts/src/admin/wrapper.html [hunk 2]
```diff
@@ -1,492 +1,510 @@
...
+
+		toastElemnt = document.querySelector("#toast");
+		previewBtnElemnt.addEventListener("click", refreshPreviewBtnHandler);
+		publishBtnElemnt.addEventListener("click", publishBtnHandler);
+
+		document.querySelectorAll(".kin2-wallet").forEach(element => previewBalance(element));
+		document.querySelectorAll(".kin3-wallet").forEach(element => previewBalance(element));
+		if ($isProduction$) {  // template variable
+			document.body.classList.add("live-production")
+		}
```
Comment (liorama) [inline scripts/src/admin/wrapper.html]: This was added

File: scripts/src/admin/wrapper.html [hunk 3]
```diff
@@ -1,492 +1,510 @@
...
+	<style>
+		body {
+			font-family: Arial;
+			background: #ffffff;
+		}
+
+		body.live-production {
+			border: 13px red inset;
+			margin: 0;
+			padding: 2px;
```
Comment (liorama) [inline scripts/src/admin/wrapper.html]: This is new

File: scripts/src/admin/wrapper.html (hunk 2 above)
Comment (doodyparizada) [reply scripts/src/admin/wrapper.html]: ;

File: scripts/src/admin/wrapper.html (hunk 1 above)
Comment (doodyparizada) [reply scripts/src/admin/wrapper.html]: ok. Didn't know about confirm. Does it work on Netscape?

**Review by doodyparizada — APPROVED** [review]

**Pull request merged**